azure-cs-cosmosapp-component/CosmosApp.cs

```diff
@@ -0,0 +1,184 @@
+﻿// Copyright 2016-2018, Pulumi Corporation.  All rights reserved.
+
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+
+using Pulumi;
+using Pulumi.Azure.Core;
+using Pulumi.Azure.Cosmosdb;
+using Pulumi.Azure.Cosmosdb.Inputs;
+using Pulumi.Azure.Trafficmanager;
+using Pulumi.Azure.Trafficmanager.Inputs;
```

Comment:
I think it doesn't make sense to put child argument types into `Inputs` namespace. I end up importing it pretty much always, and until then the types aren't visible. cc @pgavlin